internal/core/firefly_config.go

```diff
@@ -195,6 +202,23 @@ func NewFireflyConfig(stack *types.Stack, member *types.Member) *FireflyConfig {
 			},
 		},
 	}
+
+	if stack.PrometheusEnabled {
+		memberConfig.Metrics = &MetricsServerConfig{
+			HttpServerConfig: HttpServerConfig{
+				Port:      member.ExposedFireflyMetricsPort,
```

Comment:
Still need a member field for the metrics port so that we can use it when "templating" the Prometheus config, though I could see the argument for removing `Exposed` prefix since it technically isn't exposed...